src/mixpanelAnalytics.test.js

```diff
@@ -19,6 +19,7 @@ describe('MixpanelAnalytics', () => {
   });
 
   describe('#trackAssignment()', () => {
+    // eslint-disable-next-line jest/no-done-callback
     it('calls window.mixpanel.track()', done => {
```

Comment:
The fix for the eslint error, just use an anonymous callback and delete the call to`done`. 
```suggestion
    it('calls window.mixpanel.track()', () => {
```